projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-woocommerce.php

```diff
@@ -2,63 +2,21 @@
 
 use Automattic\Jetpack\Sync\Modules;
 
+require_once __DIR__ . '/../trait-woo-tests.php';
+
 /**
  * Testing WooCommerce Sync
+ *
+ * @group woocommerce
  */
 class WP_Test_Jetpack_Sync_WooCommerce extends WP_Test_Jetpack_Sync_Base {
+	use WooCommerceTestTrait;
+
 	protected $post;
 	protected $callable_module;
 	protected $full_sync;
 	public static $woo_enabled;
```

Comment:
We might want to move this `$woo_enabled` field declaration into the trait too, so every class using it doesn't have to declare it itself.